.golangci.yml

```diff
@@ -1,79 +1,124 @@
+# v1.2.0. Created based on golangci-lint v1.56.2
 
 run:
-  modules-download-mode: readonly
+  timeout: 5m
   skip-dirs-use-default: false
+  modules-download-mode: readonly
+  allow-serial-runners: true
   skip-dirs:
-    - internal
+    - internal # TODO: Do not ignore interal
 
 output:
   sort-results: true
+  uniq-by-line: false
 
 linters-settings:
-  testifylint:
-    enable-all: true
-    disable:
-      - go-require
+  depguard:
+    rules:
+      all:
+        list-mode: lax
+        deny:
+          - pkg: "flag"
+            desc: '`flag` package is only allowed in main.go'
+          - pkg: "io/ioutil"
+            desc: '`io/ioutil` package is deprecated, use the `io` and `os` package instead'
+          # TODO: Prevent using these without a reason
+          # - pkg: "reflect"
+          #   desc: '`reflect` package is dangerous to use'
+          # - pkg: "unsafe"
+          #   desc: '`unsafe` package is dangerous to use'
+
   errcheck:
-     check-type-assertions: true
-     check-blank: true
-     disable-default-exclusions: true
-     exclude-functions:
-       - '(*bytes.Buffer).Write' # always returns nil error
-       - '(*github.com/valyala/bytebufferpool.ByteBuffer).Write' # always returns nil error
-       - '(*github.com/valyala/bytebufferpool.ByteBuffer).WriteByte' # always returns nil error
-       - '(*github.com/valyala/bytebufferpool.ByteBuffer).WriteString' # always returns nil error
+    check-type-assertions: true
+    check-blank: true
+    disable-default-exclusions: true
+    exclude-functions:
+      - '(*bytes.Buffer).Write' # always returns nil error
+      - '(*github.com/valyala/bytebufferpool.ByteBuffer).Write' # always returns nil error
+      - '(*github.com/valyala/bytebufferpool.ByteBuffer).WriteByte' # always returns nil error
+      - '(*github.com/valyala/bytebufferpool.ByteBuffer).WriteString' # always returns nil error
 
   errchkjson:
     report-no-exported: true
 
   exhaustive:
+    check-generated: true
     default-signifies-exhaustive: true
 
   forbidigo:
     forbid:
-      - ^(fmt\.Print(|f|ln)|print|println)$
-      - 'http\.Default(Client|Transport)'
+      - ^print(ln)?$
+      - ^fmt\.Print(f|ln)?$
+      - ^http\.Default(Client|ServeMux|Transport)$
       # TODO: Eventually enable these patterns
-      # - 'time\.Sleep'
-      # - 'panic'
+      # - ^panic$
+      # - ^time\.Sleep$
+    analyze-types: true
+
+  gci:
+    sections:
+      - standard
+      - prefix(github.com/gofiber/fiber)
+      - default
+      - blank
+      - dot
+      # - alias
+    custom-order: true
+
+  goconst:
+    numbers: true
 
   gocritic:
+    # TODO: Uncomment the following lines
+    # enabled-tags:
+    #   - diagnostic
+    #   - style
+    #   - performance
+    #   - experimental
+    #   - opinionated
     disabled-checks:
-      - ifElseChain
+      - ifElseChain # TODO: Do not disable
+      - hugeParam
+      - rangeExprCopy
+      - rangeValCopy
```

Comment:
does the above TODO also apply to these

Reply:
No, it's an inline-comment, so it only applies that that one specified line.